src/LandPlanner.cpp

```diff
@@ -118,9 +121,26 @@ bool LandPlanner::getTargetTwist(const ros::Time& time,
             return false;
         }
         else if(anyPressed(landing_gear_message_)) {
-            state_ = LandState::DONE;
+            state_ = LandState::DISARM;
         }
     }
+    else if (state_ == LandState::DISARM) {
```

Comment:
This block is using 2 space tabs. our style is 4 spaces. Please make your indents 4 spaces.

Reply:
wait are two tabs not equal to four spaces?

Reply:
nevermind figured it out

Reply:
Sublime text has a setting that turns all of your tabs to spaces. I'm not sure if that's what you found when you figured it out, but if it isn't, it's what I recommend. I can show you on Sunday.